src/theme/foundations/radius.ts

```diff
@@ -0,0 +1,8 @@
+const radii = {
```

Comment:
Should use `radius` here as well? 🤔 
```suggestion
const radius = {
```

Reply:
@hoangmirs I use this like the library did [here](https://github.com/chakra-ui/chakra-ui/blob/main/packages/theme/src/foundations/radius.ts)